sdk/ai/azure-ai-projects/azure/ai/projects/telemetry/_utils.py

```diff
@@ -63,6 +61,43 @@
 GEN_AI_REQUEST_STRUCTURED_INPUTS = "gen_ai.request.structured_inputs"
 GEN_AI_AGENT_VERSION = "gen_ai.agent.version"
 
+# Additional attribute names
+GEN_AI_CONVERSATION_ID = "gen_ai.conversation.id"
+GEN_AI_CONVERSATION_ITEM_ID = "gen_ai.conversation.item.id"
```

Comment:
I recently learned that you can mark a string as "read-only" as shown below. This means that if someone accidently tries to assign a value to it in the code, some quality gate tool (no sure which one, you can give it a try) will fail. Please consider using it. I'll do so in any new code I write. Or will update old code. You can do this in the next PR.

```python
from typing import Final

CONSTANT_STRING: Final[str] = "Hello World"
```